Tests/SwiftProtobufTests/Test_JSON.swift

```diff
@@ -653,21 +666,206 @@ class Test_JSON: XCTestCase, PBTestHelpers {
 class Test_JSONPacked: XCTestCase, PBTestHelpers {
     typealias MessageTestType = Proto3TestPackedTypes
 
+    func testPackedFloat() {
+        assertJSONEncode("{\"packedFloat\":[1]}") {(o: inout MessageTestType) in
```

Comment:
Didn't float and double have different code paths for non integer values?  Do you want coverage for those also?